libs/ledger-live-common/src/market/v2/useMarketDataProvider.ts

```diff
@@ -0,0 +1,128 @@
+import { useQueries, useQuery } from "@tanstack/react-query";
```

Comment:
`useMarketDataProvider.ts` ... is this to query `crypto and fiat currencies conversion rates`? could it be more semantical?

Reply:
No we don't fetch conversion rates here, I just retook same name as we had before